<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>affectation</class>
 <widget class="QDialog" name="affectation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="afficher_affectation">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical {
                 background: #0094ff;
                }
                QScrollBar:horizontal {
                 background: #0094ff;
                }

QTableView  
{  
  qproperty-currentItemBackground: #cccccc;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cx_id_affectation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
padding: 5px;
border-radius: 5px;

font-weight: bold;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Id affectation :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>310</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;CIN :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>365</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Id Matériel :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>640</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:#0094ff;
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
   </property>
   <property name="text">
    <string>Ajouter </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:#0094ff;
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
   </property>
   <property name="text">
    <string>Afficher </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
padding: 5px;
border-radius: 5px;

font-weight: bold;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_materiel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
padding: 5px;
border-radius: 5px;

font-weight: bold;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:#0094ff;
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
   </property>
   <property name="text">
    <string>Supprimer </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:#0094ff;
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>330</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { border: none;image :url(C:/Users/ASUS/Downloads/loupe.png);}
QPushButton:hover
{
border: none;image :url(C:/Users/ASUS/Downloads/business-and-finance (3).png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Recherche">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
padding: 5px;
border-radius: 5px;

font-weight: bold;
}</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>250</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Affectation ↑</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>250</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Materiel ↑</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Affectation ↓</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>290</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Materiel ↓</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No Action ....</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Trier Selon :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#0000ff;&quot;&gt;Mahrous ++&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>pic</zorder>
  <zorder>afficher_affectation</zorder>
  <zorder>cx_id_affectation</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>comboBox_cin</zorder>
  <zorder>comboBox_materiel</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>Recherche</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
